drivers/syslog/Kconfig

```diff
@@ -286,6 +286,13 @@ config SYSLOG_FILE_ROTATE
 		This option is useful to ensure that log files do not get
 		huge after prolonged periods of system operation.
 
+config SYSLOG_FILE_ROTATIONS
+	int "Log rotations"
+	default 1
```

Comment:
we can remove SYSLOG_FILE_ROTATE and set SYSLOG_FILE_ROTATIONS = 0 to disable rotation.

Reply:
done